Homework/Catalog.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Text.Json;
+
+namespace Homework
+{
+    internal class Catalog
+    {
+        internal List<Product> Products { get; }
+
+        internal Catalog(List<Product> products) 
+        {
+            Products = products;
+        }
+
+        internal string produktsSerializeJson(List<Product> products) 
+        {
+            List<Product> produktNaSerializaciu = new List<Product>();
+            produktNaSerializaciu = products.ToList();
+            string json = JsonSerializer.Serialize(produktNaSerializaciu);
+            Console.WriteLine("Serializovany objekt:");
+            Console.WriteLine(json);
+            return json;
```

Comment:
toto je zvlastni, sice zadani bylo mysleno jinak, ale i tak - nestacilo by tady toto? 
```suggestion
            string json = JsonSerializer.Serialize(products);
            Console.WriteLine("Serializovany objekt:");
            Console.WriteLine(json);
            return json;
```